src/Menu/HomePageRenderer.cpp

```diff
@@ -0,0 +1,449 @@
+		ImGui::SetCursorPosX((windowSize.x - iconSize.x) * 0.5f);
+
+		// Push style to remove border
+		ImGui::PushStyleVar(ImGuiStyleVar_FrameBorderSize, 0.0f);
+		ImGui::PushStyleColor(ImGuiCol_Button, ImVec4(0, 0, 0, 0));                     // Transparent background
+		ImGui::PushStyleColor(ImGuiCol_ButtonHovered, ImVec4(0.1f, 0.1f, 0.1f, 0.3f));  // Subtle hover
+		ImGui::PushStyleColor(ImGuiCol_ButtonActive, ImVec4(0.2f, 0.2f, 0.2f, 0.5f));   // Subtle click
+
+		if (ImGui::ImageButton("##DiscordButton", menu->uiIcons.discord.texture, iconSize)) {
+			ShellExecuteA(NULL, "open", "https://discord.com/invite/nkrQybAsyy", NULL, NULL, SW_SHOWNORMAL);
+		}
+
+		// Pop the style changes
+		ImGui::PopStyleColor(3);
+		ImGui::PopStyleVar();
+
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Join Community Shaders Discord Server");
+		}
+	} else {
+		// Fallback centered button when Discord icon is not available
+		float buttonWidth = 200.0f;
+		ImGui::SetCursorPosX((windowSize.x - buttonWidth) * 0.5f);
+		if (ImGui::Button("Join Discord Server", ImVec2(buttonWidth, 0))) {
+			ShellExecuteA(NULL, "open", "https://discord.com/invite/nkrQybAsyy", NULL, NULL, SW_SHOWNORMAL);
+		}
+		if (ImGui::IsItemHovered()) {
+			ImGui::SetTooltip("Join Community Shaders Discord Server - Icon not found, using fallback button");
+		}
+	}
+
+	ImGui::PopStyleVar();
+}
+
+void HomePageRenderer::RenderQuickLinksSection()
+{
+	// Quick Links title - centered
+	ImVec2 windowSize = ImGui::GetWindowSize();
+	ImVec2 titleSize = ImGui::CalcTextSize("Quick Links");
+	ImGui::SetCursorPosX((windowSize.x - titleSize.x) * 0.5f);
+	ImGui::Text("Quick Links");
+
+	// Center the button layout
+	float buttonWidth = 180.0f;
+	float totalWidth = buttonWidth * 3 + ImGui::GetStyle().ItemSpacing.x * 2;  // 3 buttons with spacing
+	ImGui::SetCursorPosX((windowSize.x - totalWidth) * 0.5f);
+
+	// External links in a row
+	if (ImGui::Button("Nexus Mods", ImVec2(buttonWidth, 0))) {
+		ShellExecuteA(NULL, "open", "https://www.nexusmods.com/skyrimspecialedition/mods/86492", NULL, NULL, SW_SHOWNORMAL);
+	}
+
+	ImGui::SameLine();
+	if (ImGui::Button("GitHub Repository", ImVec2(buttonWidth, 0))) {
+		ShellExecuteA(NULL, "open", "https://github.com/doodlum/skyrim-community-shaders", NULL, NULL, SW_SHOWNORMAL);
+	}
+
+	ImGui::SameLine();
+	if (ImGui::Button("GitHub Wiki", ImVec2(buttonWidth, 0))) {
+		ShellExecuteA(NULL, "open", "https://github.com/doodlum/skyrim-community-shaders/wiki", NULL, NULL, SW_SHOWNORMAL);
+	}
+}
+
+void HomePageRenderer::RenderFAQSection()
+{
+	// FAQ title - centered
+	ImVec2 windowSize = ImGui::GetWindowSize();
+	ImVec2 titleSize = ImGui::CalcTextSize("Frequently Asked Questions");
+	ImGui::SetCursorPosX((windowSize.x - titleSize.x) * 0.5f);
+	ImGui::Text("Frequently Asked Questions");
+	ImGui::Separator();
+
+	// FAQ items with collapsible headers
+	if (ImGui::CollapsingHeader("What is Community Shaders?")) {
```

Comment:
Do we really need a 3rd location for a FAQ? Particularly one that hard codes text so we have to do a new release to fix a typo?